plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestingMemSqlServer.java

```diff
@@ -49,6 +49,7 @@ public TestingMemSqlServer(String dockerImageName)
                 // Lower the size of pre-allocated log files to 1MB (minimum allowed) to reduce disk footprint
                 "memsql-admin update-config --yes --all --set-global --key \"log_file_size_partitions\" --value \"1048576\" && " +
                 "memsql-admin update-config --yes --all --set-global --key \"log_file_size_ref_dbs\" --value \"1048576\" && " +
+                // re-execute startup to actually start the nodes (first run performs setup but doesn't start the nodes)
```

Comment:
`/startup && .... && /startup` looks very quirky.

I took a look at what the script does and it's sad that /startup seems to have no explicit control for that....